src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -271,7 +271,7 @@ void EIRPlantLoopHeatPump::setOperatingFlowRatesASHP(EnergyPlusData &state)
                                              this->loadSideLocation.compNum);
 
         // if there's no flow in one, try to turn the entire heat pump off
-        if (this->loadSideMassFlowRate <= 0.0 || this->sourceSideMassFlowRate <= 0.0) {
+        if (this->loadSideMassFlowRate <= 0.0) {
```

Comment:
I was initially surprised by this change, but I see the function scope is only for ASHP, so this is good to go.